include/deal.II/base/vectorization.h

```diff
@@ -169,6 +171,29 @@ class VectorizedArrayIterator
     return *this;
   }
 
+  /**
+   * This operator advances the iterator by @p offet lanes and returns a
+   * reference to <tt>*this</tt>.
+   */
+  VectorizedArrayIterator<T> &
+  operator+=(unsigned int offset)
+  {
+    lane += offset;
+    return *this;
+  }
+
+  /**
+   * Prefix <tt>--</tt> operator: <tt>--iterator</tt>. This operator advances
+   * the iterator to the previous lane and returns a reference to
+   * <tt>*this</tt>.
+   */
+  VectorizedArrayIterator<T> &
+  operator--()
+  {
+    lane--;
+    return *this;
```

Comment:
That would suggest also putting an assertion here. (And possibly in `operator++`, which is not part of this patch.)